docs/getting_started/Install.md

```diff
@@ -0,0 +1,108 @@
+# Installation
+
+The following will walk through the deployment of Pelorus.
+
+## Prerequisites
+
+Before deploying Pelorus, the following tools are necessary
+
+* A Unix machine from which to run the install (usually your laptop)
+* An OpenShift 4.7 or higher environment
+* [git](https://git-scm.com/)
+* [oc](https://docs.openshift.com/container-platform/4.8/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli) The OpenShift CLI**\***
+* [helm](https://helm.sh/) CLI 3 or higher**\***
+
+>**Note:** It is possible to install `oc` and `helm` inside a Python virtual environment. To do so, change to Pelorus directory (after cloning its repository), and run
+>```
+>make dev-env
+>source .venv/bin/activate
+>```
+
+## Initial Deployment
+
+To begin, clone Pelorus repository. To do so, you can run
+```
+git clone https://github.com/konveyor/pelorus.git
+```
+which will download the latest code of Pelorus.
+
+To download a stable version, run
+```
+git clone --depth 1 --branch <TAG> https://github.com/konveyor/pelorus.git
+```
+changing **TAG** by one of [Pelorus versions](https://github.com/konveyor/pelorus/tags).
+
+Change the current directory to `pelorus`, by running
+```
+cd pelorus
+```
+
+Login to the OpenShift cluster as **kube:admin** (the specific user is really necessary?), by running
+```
+oc login
+```
+
+Create the namespace called `pelorus` that Pelorus will use, by running
+```
+oc create namespace pelorus
+```
+You can choose any name you wish, but remember to change to the same name in the following commands.
+
+Pelorus gets installed via helm charts. First, deploy the operators on which Pelorus depends, by running
+```
+helm install operators charts/operators --namespace pelorus
+```
+
+Wait for the operators install to complete. (create script that checks when is finished https://github.com/hjwp/book-example/blob/master/functional_tests/base.py#L17 and I would change to an image)
```

Comment:
We already have that script: `run-pelorus-e2e-tests`!

Can I remove the part of installation from that script and make a separate script just for installation?